schema/frogbot-schema.json

```diff
@@ -219,6 +219,11 @@
               "type": "string",
               "title": "Virtual Artifactory Repository",
               "description": "Name of a Virtual Repository in Artifactory to resolve (download) the project dependencies from"
+            },
+            "uniqueIdentifier": {
```

Comment:
change field name to "uniqueName"

Reply:
`Name` is also a good choise for the field name

Reply:
changed to projectName